crates/async-std-resolver/src/tests.rs

```diff
@@ -1,7 +1,7 @@
 #![allow(clippy::extra_unused_type_parameters)]
 
 use hickory_resolver::name_server::GenericConnection;
-use hickory_resolver::testing;
+use hickory_resolver::testing::*;
```

Comment:
I've been trying to get rid of all the `*` imports (except for the `super::*` ones in inline `test` modules), would prefer to name the necessary symbols here.